wherescape/connectors/hubspot/hubspot_wrapper.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import hubspot
+
+from hubspot.crm.properties import ApiException
+from .send_data import (
+    send_company_object,
+    send_contact_object,
+    send_deal_object,
+)
+
+"""
+module that makes sure the data is prepared correctly depending on where and how it will be send.
+Also provides the properties depending on the object.
+"""
+
+
+class Hubspot:
+    def __init__(self, access_token):
+        self.access_token = access_token
+        self.client = hubspot.Client.create(access_token=access_token)
+
+    def get_company_properties(self):
+        """
+        this method returns all information of company properties
+        """
+        logging.info("getting properties of companies")
+        return self.get_object_properties("companies")
+
+    def get_deal_properties(self):
+        """
+        this method returns all information of deal properties
+        """
+        logging.info("getting properties of deals")
+        return self.get_object_properties("deals")
+
+    def get_contact_properties(self):
+        """
+        this method returns all information of contact properties
+        """
+        logging.info("getting properties of contacts")
+        return self.get_object_properties("contacts")
+
+    def send_company_patch(self, **args):
+        """
+        method for sending patches for companies
+        """
+        inputs = args["inputs"]
+
+        if len(inputs) == 1:
+            id = inputs[0]["id"]
+            properties = inputs[0]["properties"]
+            send_company_object.patch_company_on_id(id, properties, self.client)
+        elif len(inputs) > 1:
+            send_company_object.patch_company_batch(inputs, self.client)
+
+    def send_contact_patch(self, **args):
+        """
+        method for sending patches for contacts
+        """
+        inputs = args["inputs"]
+
+        if len(inputs) == 1:
+            id = inputs[0]["id"]
+            properties = inputs[0]["properties"]
+            send_contact_object.patch_company_on_id(id, properties, self.client)
+        elif len(inputs) > 1:
+            send_contact_object.patch_company_batch(inputs, self.client)
+
+    def send_deal_patch(self, **args):
```

Comment:
I see some repetition in the three functions, like: send_contact_patch
It can be one function! I also see that for send_contact_patch you used the function patch_company_on_id. I believe it should be patch_on_id, since send_contact_object doesn't expose such a function.